src/pages/en/core-concepts/project-structure.md

```diff
@@ -23,30 +25,29 @@ Astro leverages an opinionated folder layout for your project. Every Astro proje
 
 A common Astro project directory might look like this:
 
-```
-├── src/
-│   ├── components/
-│   │   ├── Header.astro
-│   │   └-─ Button.jsx
-│   ├── layouts/
-│   │   └-─ PostLayout.astro
-│   └── pages/
-│   │   ├── posts/
-│   │   │   ├── post1.md
-│   │   │   ├── post2.md
-│   │   │   └── post3.md
-│   │   └── index.astro
-│   └── styles/
-│       └-─ global.css
-├── public/
-│   ├── robots.txt
-│   ├── favicon.svg
-│   └-─ social-image.png
-├── astro.config.mjs
-├── package.json
-└── tsconfig.json
-
-```
+<FileTree>
+- public/
+  - robots.txt
+  - favicon.svg
+  - social-image.png
+- src/
+  - components/
+    - Header.astro
+    - Button.jsx
+  - layouts/
+    - PostLayout.astro
+  - pages/
+    - posts/
+      - post1.md
+      - post2.mdx
```

Comment:
Change from when I was testing snuck in. Should change back but also making a note for tomorrow to add a file association/icon for MDX

```suggestion
      - post2.md
```